www/front_src/src/Resources/Actions/Acknowledge/index.tsx

```diff
@@ -101,4 +101,8 @@ const AcknowledgeForm = ({
   );
 };
 
-export default AcknowledgeForm;
+export default React.memo(
```

Comment:
We should avoid rendering the Form at all when there is nothing to be displayed instead